modules/launcher/src/main/java/org/mule/runtime/module/launcher/artifact/ArtifactMuleContextBuilder.java

```diff
@@ -199,6 +218,7 @@ public MuleContext build() throws ConfigurationException
         checkState(executionClassLoader != null, EXECUTION_CLASSLOADER_WAS_NOT_SET);
         checkState(APP.equals(artifactType) || parentContext == null, ONLY_APPLICATIONS_ARE_ALLOWED_TO_HAVE_A_PARENT_CONTEXT);
         List<ConfigurationBuilder> builders = new LinkedList<>();
+        builders.add(new ServicesConfigurationBuilder(serviceRepository));
```

Comment:
This line goes away once you register the services using SpringMuleContextServiceConfigurator